vars/detectDockerAgent.groovy

```diff
@@ -4,9 +4,18 @@ Map call(Map args = [:]) {
     def inputManifest = lib.jenkins.InputManifest.new(readYaml(file: manifest))
     dockerImage = inputManifest.ci?.image?.name ?: 'opensearchstaging/ci-runner:ci-runner-centos7-v1'
     dockerArgs = inputManifest.ci?.image?.args
+    // Using default javaVersion as openjdk-17
+    String javaVersion = 'openjdk-17'
+    java.util.regex.Matcher jdkMatch = (dockerArgs =~ /openjdk-\d+/) 
+    if (jdkMatch.find()) {
+        def jdkMatchLine = jdkMatch[0]
+        javaVersion = jdkMatchLine
+    }
```

Comment:
Seems complicated :) Match any 1+ digit

Reply:
Hey @dblock I have added the parsing logic, assuming we add multiple args in future to the manifest file
```
args: -e JAVA_HOME=/opt/java/openjdk-17 -e PATH=<> -p 8080:8080
```